safenode/src/bin/kadclient.rs

```diff
@@ -138,6 +145,42 @@ async fn wallet(opt: &Opt) -> Result<()> {
         }
     }
 
+    if let Some(hex) = &opt.send_to {
+        let address = parse_public_address(hex)?;
+
+        if let Some(amount) = opt
+            .send_amount
+            .clone()
+            .map(|amount| parse_tokens_amount(&amount))
+        {
+            if amount.as_nano() > 0 {
```

Comment:
Same here perpaps, show user an error or message if they specify `0`. I am also in favour of handling such cases with an early return to make sure we keep the code indentation low.

Reply:
Probably good to check the next one to see what stuff from here, is fixed there.